src/schematools/datasetcollection.py

```diff
@@ -24,29 +24,23 @@ class DatasetCollection(metaclass=Singleton):
     """
 
     def __init__(self) -> None:
-        """Initialize the DatasetCollection.
-
-        Args:
-            schema_loader: An alternative schema loader can be provided
-                If schema_loader is None, the default url loader is used.
-        """
+        """Initialize the DatasetCollection."""
```

Comment:
Can we remove this docstring without tripping up flake8? Because this one is just visual noise.

```suggestion
```

Reply:
Yes, totally agree, constructor without args does not need a docstring. I added a noqa for it to silence the flake8 check.